### ISSUE TITLE
Bitdefender: "suspicious connection blocked"

### ISSUE THREAD

**Issue by foreachthing** [cnwangjie/better-onetab#329]
扩展版本 (extension version): 1.4.7
浏览器版本 (browser version):  85.0.4183.102 (Official Build) (64-bit)
操作系统 (operating system): Win10x64
问题描述 (problem description):

Sorry, got this message from Bitdefender:
![image](https://user-images.githubusercontent.com/10420187/92995016-d6743800-f4ff-11ea-89bb-2de68310e0b0.png)

and removed your extension.
Please fix your certificate and I'm back using it.

**Comment by glhey** [comment]
Yep, agreed - this is a great extension - applause, so appreciate update your certificate!
